uwrt_mars_rover_hw_bridge/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
how did u come up with this version number? should be at least the latest version that comes with ubuntu 18.04.